test/e2e/support/utils.js

```diff
@@ -81,6 +108,13 @@ export async function getHTMLFromCodeEditor() {
 	return textEditorContent;
 }
 
+export async function clearLocalStorage() {
+	await page.$eval( 'body', ( body ) => {
```

Comment:
We should be able to use `page.evaluate` here, which is run as within the context of the browser:

```js
await page.evaluate( () => window.localStorage.clear() );
```

Reply:
Hi @aduth, thank you I applied this version and it works great.
Before I tried it and I had problems but I think the problem was related to other error that was meanwhile solved because now it works exactly as expected.